File: js/1071.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

x=parseInt(lines[0])
y=parseInt(lines[1])
cont=0
for (i=y+1;i<x;i++){
    if (i%2!==0){
        cont+=i
    }
}
console.log(cont +'')

File: c/1071.c
#include <stdio.h>
int main(){
    int x, y, i, cont=0;
    scanf("%d %d", &x, &y);

    for (i=y+1;i<x;i++){
        if (i%2 != 0){
            cont+=i;
        }
    }
    printf("%d\n", cont);
}